drake/systems/plants/RigidBodyTree.cpp

```diff
@@ -1326,6 +1326,31 @@ shared_ptr<RigidBody> RigidBodyTree::findLink(std::string linkname, int robot) c
   return nullptr;
 }
 
+shared_ptr<RigidBodyFrame> RigidBodyTree::findFrame(std::string frame_name, int robot) const
```

Comment:
okay, I've switched to using the model name instead of robot number for findFrame